app/src/main/res/values-it/strings.xml

```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="launch_dialog_content">Benvenuto su Extras for Android Auto, un tema per cambiare l\'aspetto di Android Auto
-        \n\n Se vuoi ringraziarmi per il lavoro, puoi farlo con una donazione, dal bottone sotto.
+    <string name="previews_text">🖼 Anteprime! 🖼</string>
+    <string name="launch_dialog_title" translatable="false">Ciao!</string>
```

Comment:
This string is not translatable

Reply:
What string= Hey?

Reply:
Yes. String "launch_dialog_title" is deliberately not translatable